ion-tree-list.js

```diff
@@ -18,10 +18,10 @@ function addDepthToTree(obj, depth, collapsed) {
 }
 
 function toggleCollapse(obj) {
-    for (var key in obj) {
-        if (obj[key] && typeof(obj[key]) == 'object') {
-            obj[key].collapsed = !obj[key].collapsed;
-            toggleCollapse(obj[key])
+    if (obj.tree) {
```

Comment:
hi @yesir1006 ! thank you very much for contributing! 

Do you think it is possible to work on a solution where we consider more than one level depth?